packages/docs/docs/docs/using.md

```diff
@@ -1,18 +1,18 @@
 ---
 id: using
-title: Using the stdlib in your app
-sidebar_label: Using the stdlib
+title: Using the EVM package in your app
```

Comment:
What about "Using EVM packages in your app"?

Reply:
Good catch. Something I missed when I systematically searched and replaced. I fixed this one and looked for other situations where the same error could have occurred and actually found a few.